apps/masterbots.ai/components/routes/chat/chat-options.tsx

```
@@ -1,26 +1,26 @@
   Eye,
   EyeOff,
   MoreVertical,
-  Trash,
-  BadgeCheck
+  Trash
 } from 'lucide-react'
-import { useThreadVisibility } from '@/lib/hooks/use-thread-visibility'
-import React, { useState } from 'react'
 import type { Thread } from 'mb-genql'
 import { toSlug } from 'mb-lib'
+import React, { useState } from 'react'
```

Biome

[error] The default import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)